### PR TITLE
DRAFT: Updates to Flash Mint Module

### PR DESCRIPTION
**Pull request by patrick-j-govalpha**
@LongForWisdom @Gala-G  made some updates to Flash Mint Module documentation if anyone has suggestions

### CODE REVIEW

File: module-index/module-flash-mint-module.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ Techincal Docs: https://docs.makerdao.com/smart-contract-modules/flash-mint-modu
 
 The Flash Mint Module allows anyone to permisionlessly mint Dai as long as it is paid back in the same transaction. This is commonly known as a Flash Loan.
 
-Users can combine Flash Loans with other smart contract functions to perform powerful multi-step transactions across multiple decentralized exchanges (DEX). This process can be used to exploit arbitrage opportunities when Dai deviates from the Peg.
+Users can combine Flash Loans with other smart contract functions to perform powerful multi-step transactions across multiple decentralized exchanges (DEX). This process can be used to exploit arbitrage opportunities when Dai deviates from the Peg, among other uses.
```
Comment (LongForWisdom) [inline module-index/module-flash-mint-module.md:14]: I would maybe rephrase some, or just leave as is. "Multi-step transactions of this nature have many uses, including arbitrage and improving user experience when creating leveraged positions."

File: module-index/module-flash-mint-module.md [hunk 2]
```diff
@@ -22,25 +22,22 @@ To demonstrate how this works in practice, we will imagine Dai trades at 1.02 DA
...
+* Market efficiency of DAI is improved by Flash Loan arbitrage - a secondary effect of this could be increased liquidity in DAI markets and Peg stability as deviations from the Peg will be more susceptible to arbitrage.
+* The ability to borrow large sums increases the utility of Flash Loans to identify exploits in DeFi protocols - by exposing exploits and attack vectors, the DeFi ecosystem can be made more robust.
+* The Flash Mint Module encourages further integration between the Maker Protocol and other Decentralized Apps - DEX aggregators can use it to ensure their users get the best prices available, or Vault automation systems can utilize Flash Loans to leverage and deleverage Vaults.
+* Income for The Maker Protocol - The Maker Protocol can charge Minting Fees on Flash Loans that use the Flash Mint Module. Minting Fees have the potential to be an alternative source of revenue for the Maker Protocol. Typically, fees charged on Flash Loans are in the order of 0-0.1%.
+* The Flash Mint Module is permissionless, meaning anyone can use it. Similarly, it makes access to arbitrage opportunities to be easier as it is no longer capital dependant.
```
Comment (LongForWisdom) [inline module-index/module-flash-mint-module.md:40]: I know you just copied this, but the last bullet doesn't make sense. 'it makes access to arbitrage opportunities *to be* easier as it is no longer capital *dependent*.

**Review by LongForWisdom — CHANGES_REQUESTED** [review]
I think I'm just going to make changes to this myself before we add it to the nav bar, so feel free to ignore these and I'll just fix it later.

File: module-index/module-flash-mint-module.md (hunk 2 above)
Comment (mkrorbrkr) [reply module-index/module-flash-mint-module.md:40]: I think I changed this from sentences to bullet points but made a pigs ear of it. I'm happy to adjust it later today

**Pull request merged**